cmd/sks_nodepool_update.go

```diff
@@ -150,10 +150,10 @@ func init() {
 	sksNodepoolUpdateCmd.Flags().String("instance-type", "", "Nodepool Compute instances type")
 	sksNodepoolUpdateCmd.Flags().Int64("disk-size", 0, "Nodepool Compute instances disk size")
 	sksNodepoolUpdateCmd.Flags().StringSlice("anti-affinity-group", nil,
-		"Nodepool Anti-Affinity Group <name | id> (can be specified multiple times). "+
+		"Nodepool Anti-Affinity Group NAME|ID|NAME|ID (can be specified multiple times). "+
 			"Note: this replaces the current value, it is not cumulative.")
 	sksNodepoolUpdateCmd.Flags().StringSlice("security-group", nil,
-		"Nodepool Security Group <name | id> (can be specified multiple times)"+
+		"Nodepool Security Group NAME|ID|NAME|ID (can be specified multiple times)"+
```

Comment:
Erroneous repetition?
```suggestion
		"Nodepool Security Group NAME|ID (can be specified multiple times)"+
```